portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1092,6 +1092,55 @@ definition {
 		PortletEntry.publish();
 	}
 
+	@description = "This is a use case for LPS-110450."
+	@priority = "5"
+	test VerifyErrorMessageInSaas {
+		property osgi.app.includes = "saml-saas";
+		property test.name.skip.portal.instance = "SAML#ErrorMessageValidation";
```

Comment:
This property needs to match the test name

Reply:
I update it.